pyproject.toml

```diff
@@ -395,6 +395,9 @@ doctest_optionflags = [
   "ELLIPSIS",
 ]
 filterwarnings = [
+  "error:::pandas",
```

Comment:
what does this one mean?

Reply:
If any unhanded warning originates from the path `pandas/*`, treat it as as an error in the test suite.

Any subsequent filter in this list takes prescient over this one if they overlap.